suite/tests/CMakeLists.txt

```diff
@@ -732,33 +736,38 @@ function(configure_app_api_build_flags test decoder use_static_DR)
     configure_DynamoRIO_standalone(${test})
     set(extra_flags "-DSTANDALONE")
   endif ()
-  # append to properties set by configure_DynamoRIO_standalone
+  # Since we're including configure.h we do not want any default defines.
+  get_target_property(prop_defs ${test} COMPILE_DEFINITIONS)
```

Comment:
We expect prop_defs to be a list right? Will the regex replace below work as intended?

Reply:
A list in quotes is a string with the list items separated by semicolons, so it will replace the items, possibly leaving empty slots: but that works out ok b/c the logic that turns it into "-DFOO" tokens skips the empty ones.